src/OpenTelemetry.Exporter.Zipkin/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -2,13 +2,26 @@ OpenTelemetry.Exporter.Zipkin.ZipkinExporter
 OpenTelemetry.Exporter.Zipkin.ZipkinExporterOptions
 OpenTelemetry.Exporter.Zipkin.ZipkinExporterOptions.Endpoint.get -> System.Uri
 OpenTelemetry.Exporter.Zipkin.ZipkinExporterOptions.Endpoint.set -> void
+OpenTelemetry.Exporter.Zipkin.ZipkinExporterOptions.ExporterTimeoutMilliseconds.get -> int
```

Comment:
This is actually the `TimeoutMilliseconds` for processor instead of exporter.

Reply:
It's called exporterTimeoutMilliseconds in the BatchExportProcessor ctor. Should I rename it to TimeoutMilliseconds?

Reply:
I would vote to call it `TimeoutMilliseconds` here.
@cijothomas what's your opinion?

Reply:
Anything who name starts with Exporter, and part of Exporter specific package could be misleading.
What if we create BatchProcessorOptions class in SDK, and have every exporterOption expose a setting BatchProcessorOptions.?

Reply:
Fixed this.